src/generic_actor.cr

```diff
@@ -95,10 +91,16 @@ module GenericActor
 
     def {{name}}({% if args %}*,{% for k, v in args %}{{k}} : {{v}},{% end %}{% end %}) : {{result}}
       message = {{message_type}}.new({% if args %}{ {% for k, v in args %}{{k}}: {{k}},{% end %} }{% end %})
+      check_message_loop
       @message_queue.send(message)
       message.await
     end
 
+    protected def check_message_loop
```

Comment:
Looks like this function will be generated under the `macro call_def`, but in case if class will use `cast_def` but not use `call_def` there gona be compilation error because check_message_loop will not be generated 

Propose move this function to closer to `@message_queue` and  `@message_queue` variables. 
Also good for readability